eng/native/version/copy_version_files.sh

```diff
@@ -0,0 +1,5 @@
+__VersionFolder="$(cd "$(dirname "$0")"; pwd -P)"
+__RepoRoot="$(cd "$(dirname "$__VersionFolder")/../../"; pwd -P)"
+
+# Use yes n and interactive cp instead of -n since -n is not supported on macOS.
+yes n | cp -i "${__VersionFolder}/"*{.h,.c} "$__RepoRoot/artifacts/obj/"
```

Comment:
`yes n | cp -i ..` results in exit code 1 when the file exists. This is causing rebuild of corehost to fail because we have `set -e` in its build.sh. Instead, we can use regular syntax:

```suggestion
for path in "${__VersionFolder}/"*{.h,.c}; do
    # -s <filename> checks if file has nonzero size (we don't need any more guarantee than that)
    if [ ! -s "$__RepoRoot/artifacts/obj/$(basename path)" ]; then
        cp "$path" "$__RepoRoot/artifacts/obj/"
    fi
done
```